Signal/src/ViewControllers/AttachmentApprovalViewController.swift

```diff
@@ -155,7 +155,7 @@ class AttachmentApprovalViewController: OWSViewController {
                                     backgroundColor:color,
                                     width:buttonWidth,
                                     height:buttonHeight,
-                                    target:target,
+                                    target:self,
```

Comment:
The crazy thing is that this only appears to crash on iOS 8.  I don't know why, but this works on iOS 11.